### PR TITLE
julia and libcxxwrap-julia: add new versions

### PR DESCRIPTION
**Pull request by melven**
Just updated versions; both build fine for me.
I didn't perform more detailed functionality checks, though.

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Confirmed SHAs. Looks good to me. Waiting on final approval from @eloop, @giordano, @haampie, or @vchuravy. Thanks @melven!

**Review by giordano — APPROVED** [review]

**Review by eloop — APPROVED** [review]

**Pull request merged**
